client/src/Component/AppHeader/AppHeader.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    flexGrow: 1,
+  },
+  appHeader: {
+    backgroundColor: '#5a3796',
+  },
+  title: {
+    flexGrow: 1,
+  },
+}));
+
+const AppHeader = () => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <AppBar position='static' className={classes.appHeader}>
+        <Toolbar>
+          <Typography variant='h6' className={classes.title}>
+            Fluxo
+          </Typography>
+          <Button color='inherit'>Login</Button>
```

Comment:
Create a generic button so that all the apps/site button are generic/same. This is helpful when you want to save same form submit/ action buttons through out the site.

Reply:
I have created a generic button in coming PR i.e login page because in this PR, I was not sure about all props that material UI supports. So, I just setup the basic button. But now in our project, all buttons are generic.
should I merge that branch in this or continue as it?